src/containers/BooksList.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Book from '../components/Book';
+import { changeFilter, removeBook } from '../actions';
+import CategoryFilter from '../components/CategoryFilter';
+
+
+const filteredBooks = (books, filter) => {
+  if (filter === 'All') {
+    return books;
+  }
+  return books.filter((book) => book.category === filter);
+};
+
+const BooksList = (props) => {
+  const {
+    changeFilter, removeBook, filter,
+  } = props;
+
+
+  const handleFilterChange = (event) => changeFilter(event.target.value);
+  const handleBookRemove = (book) => removeBook(book);
```

Comment:
Hi guys I think we still are "calling" `removeBook` in `BookList`. Let me remind that the only thing `BookList` needs to do with `removeBook` is to pass it to `Book` component.

Reply:
We are not sure what to do with this. The remove button is being passed as a `prop` from the `Book` component and the `action` is being done in the `BooksList` component.